src/commands/index.js

```diff
@@ -61,7 +61,28 @@ const ignoreChannel = {
   admin: true,
 };
 
-const commands = [helloWorld, ignore, reminder, birthday, ignoreChannel];
+const roll = {
+  name: "roll",
+  action: async (message) => {
+    const args = message.content.toLowerCase().split(" ");
+    const diceNumbers = args[1].split("d").map(nb => Number(nb));
+    if (args[1] && !isNaN(diceNumbers[0]) && !isNaN(diceNumbers[1])) {
+      let total = 0;
+      for (let i = 0; i < diceNumbers[0]; i++) {
```

Comment:
WHile we are here, you can also fix this and use a reduce instead of a for .. in